etc/kayobe/compute.yml

```
@@ -5,6 +5,7 @@
 # User with which to access the computes via SSH during bootstrap, in order
 # to setup the Kayobe user account.
 #compute_bootstrap_user:
+compute_bootstrap_user: root
```

```suggestion
compute_bootstrap_user: centos
```
centos is default user for generic cloud image